storage/oqgraph/ha_oqgraph.cc

```diff
@@ -1099,6 +1099,10 @@ int ha_oqgraph::info(uint flag)
 
 int ha_oqgraph::extra(enum ha_extra_function operation)
 {
+  if (graph->get_thd() != current_thd) {
```

Comment:
Hi @svoj thanks for the suggestion, TBH I never quite got a handle on all of the interfaces when I ported this code from mariadb5 and had to muddle through
I wont be able to push this until next week though I'm away atm